interp/newexecute_test.go

```diff
@@ -110,6 +111,33 @@ func TestResetRand(t *testing.T) {
 	}
 }
 
+func TestGetArrayValue(t *testing.T) {
+	interpreter := newInterp(t, `
+BEGIN { Arr["key"]; f(); g(Arr) }
+function f() { Arr["hello"]="world" }
+function g(arr) { arr["a"]=1.23 }`)
+	_, err := interpreter.Execute(nil)
+	if err != nil {
+		t.Fatalf("error executing: %v", err)
+	}
+	arr := interpreter.Array("Arr")
+	if len(arr) != 3 {
+		t.Errorf("wrong Arr len: %d", len(arr))
```

Comment:
Can you please change this failure output to match other similar tests: `t.Errorf("expected length 3, got %d", len(arr))`. Similar below (though use `%q` for strings).